packages/components/src/components/omnibox/OmniBox.spec.tsx

```diff
@@ -98,14 +111,14 @@ describe('OmniBox component', () => {
     const actions = [new HelloWorldAction()];
 
     test('requires only an action', () => {
-        const component = shallow(<OmniBox getModel={getModel} actions={actions} />);
+        const component = shallow(<OmniBox getColumns={getColumns} getSelectDistinctOptions={getSelectDistinctOptions} actions={actions} />);
```

Comment:
After investigating the typings for enzyme it looks like they provide a useful generic's template for both `shallow` and `mount`.

```tsx
shallow<Component>(<Component ... />);
// or
mount<Component>(<Component ... />);
```

Using these should allow for the rest of the usage to be easily typed without having to cast or left-hand define.

Reply:
Nice!